wordpress-mobile/AztecEditor-iOS
# Correct media attachment width

Fixes #796 

This is a clone of this PR https://github.com/wordpress-mobile/AztecEditor-iOS/pull/797 with conflicts sorted and some code simplication.

To test:
 - Start the demo app
 - Open the empty demo
 - Add an image object that takes the full width of the editor
 - Activate blockquote for that paragraph that has media
 - Check that the image stays inside the blockquote background.

Kudos to @azone !
